alerting/src/aa-storm-alert/alert.ts

```diff
@@ -138,34 +139,35 @@ export function transformReportsToLastProcessed(
   return lastProcessedReports;
 }
 
+// TODO - make sure spelling is exactly the same for districts as in the report
+// or add a mapping below if needed
+const districtNameMapping: { [key: string]: string } = {
```

Comment:
Check with Valentin once the data has been updated on their end.
cc @valpesendorfer